site.yml

```diff
@@ -11,6 +11,9 @@ Plugins:
       apiKey: "ALGOLIA_TOKEN"
       index: "INDEX_NAME"
   - Google Analytics: "UA-XXXXX-Y"
+  - Footer Widget:
```

Comment:
We should call it slack.  Rushenn if this is not in the site.yml, there is no need to show the widget.

Reply:
I think people might also want to link the footer widget to other sites than Slack (e.g. adding a mailto link)

Reply:
We could. But we could also be opinionated.

Reply:
We don't have to block on this @rushenn

Reply:
@deekoder 
I'll change the label to 'Slack'. This widget is conditional. Footer widget will only render if the parameters are set.